mass/adapters/outbound/utils.py

```diff
@@ -15,14 +15,18 @@
 #
 
 """Utility functions for building the aggregation pipeline used by query handler"""
-from collections import defaultdict
+from collections import OrderedDict, defaultdict
 from typing import Any, Optional
 
 from hexkit.custom_types import JsonObject
 
 from mass.core import models
 
-SORT_ORDER_CONVERSION = {"ascending": 1, "descending": -1}
+SORT_ORDER_CONVERSION: JsonObject = {
+    "ascending": 1,
+    "descending": -1,
+    "relevance": {"$meta": "textScore"},
```

Comment:
This delegates to the underlying mongo implementation of relevance scoring, right?

Reply:
Yes, it does indeed.